src/structures/Message.js

```diff
@@ -574,6 +574,11 @@ class Message extends Base {
    * @param {MessageOptions|MessageAdditions} [options] The additional options to provide
    * @param {MessageResolvable} [options.replyTo=this] The message to reply to
    * @returns {Promise<Message|Message[]>}
+   * @example
+   * // Reply to a message
+   * message.reply('This is a reply!')
+   *   .then(msg => console.log(`Replied to message "${msg.content}"`))
```

Comment:
Is this the right info to log?

Reply:
this would log `Replied to message "This is a reply!"`, is that intentional?

Reply:
That was what I was intending, although I wasn't sure if it was the right thing to log

Reply:
Fixed in ad778e8e046f2abec3bb1a90ad20a695359551e0